src/semantic/typography.tokens.json5

```diff
@@ -98,6 +98,11 @@
         value: "{core.typography.line-spacing.sm}",
         type: "number",
       },
+      moderate: {
+        comment: "",
```

Comment:
should there be a comment here like in the other tokens?

Reply:
Ideally, yes. @JoschaIco what do you think?